frame_id_fixed: greenhouse
frame_id_slammap: map
# Stop ICP if the transformation from the last iteration was this small.
icp_tf_epsilon: 0.0000000001

# During ICP, two points won't be considered a correspondence if they are at
# least this far from one another.
icp_corr_dist: 2.0

# Iterate ICP this many times.
icp_iterations: 50

# prior_map_path: /home/wvuirl/pollination_ws/src/priormap_localization/gicp_localization/config/AERB_222.pcd
# prior_map_path: /home/wvuirl/pollination_ws/src/priormap_localization/gicp_localization/config/AERB_LOBBY.pcd
prior_map_path: /home/bramblebee/pollination_ws/src/priormap_localization/gicp_localization/config/Greenhouse_corner_original_1.pcd
slam_map_topic: /laser_cloud_map
